src/search/identity_search.rs

```diff
@@ -39,7 +39,7 @@ pub fn identity_search(
                 smiles_field,
                 fingerprint_field,
                 extra_data_field,
-                &searcher,
+                searcher,
```

Comment:
Why the switch from a reference to pass by value?

Reply:
Because it was already a reference, so I was actually passing in a `&&Searcher`. Kudos to clippy for finding it :)